src/intrinsics/fb-www/react-mocks.js

```diff
@@ -506,6 +506,10 @@ export function createMockReact(realm: Realm, reactRequireName: string): ObjectV
     0,
     (context, [type, config, ...children]) => {
       invariant(type instanceof Value);
+      // if config is undefined/null, use an empy object
+      if (config === realm.intrinsics.undefined || config === realm.intrinsics.null || config === undefined) {
```

Comment:
I find it a bit confusing we check both for intrinsics and for actual undefined.

Reply:
If you look at the test I added, you'll see the 3 possible cases in there.